.github/workflows/ci-test-go.yml

```diff
@@ -107,15 +107,17 @@ jobs:
         run: |
             go install gotest.tools/gotestsum@latest
             make test-unit
-            
-      - name: Analyze with SonarCloud
-        if: ${{ github.ref_name == 'main' && github.repository_owner == 'testcontainers' && inputs.run-tests && !inputs.rootless-docker && inputs.project-directory == '.' }}
-        uses: sonarsource/sonarcloud-github-action@master
+
+      - name: Upload SonarCloud files
+        if: ${{ inputs.run-tests && !inputs.rootless-docker }}
+        #if: ${{ github.ref_name == 'main' && github.repository_owner == 'testcontainers' && inputs.run-tests && !inputs.rootless-docker }}
+        uses: actions/upload-artifact@v3
         with:
-          projectBaseDir: ${{ inputs.project-directory }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          name: sonarcloud
+          path: |
+            ./.gitignore
+            ${{ inputs.project-directory }}/unit-tests.xml
```

Comment:
Is the name of the XML file `unit-tests.xml` or `TEST-unit.xml`?

Reply:
Never mind, I just saw https://github.com/testcontainers/testcontainers-go/pull/1614/files#diff-f18f81daf1f8305a66be78cac5d122d165f39391f0348477f133858018585cbcR19